.vscode/launch.json

```diff
@@ -32,7 +32,13 @@
 			"stopOnEntry": false,
 			"sourceMaps": true,
 			"outFiles": ["${workspaceRoot}/out/test/**/*.js"],
-			"preLaunchTask": "Build"
+			"preLaunchTask": "Build",
+			"skipFiles": [
```

Comment:
`skipFiles`? It's VSCode's version of "Just My Code"

Reply:
I needed it to debug things, but it's useful in general. Otherwise if you do something like set "break on all exceptions" you be reading through `fs` code all day